gradle/versions.gradle

```diff
@@ -80,15 +80,19 @@ dependencyManagement {
     dependency 'org.rocksdb:rocksdbjni:5.15.10'
 
     dependency 'org.slf4j:slf4j-log4j12:1.7.26'
-    
+
     dependency 'org.springframework.security:spring-security-crypto:5.1.5.RELEASE'
 
     dependency 'org.web3j:abi:4.3.1'
     dependency 'org.web3j:core:4.3.1'
     dependency 'org.web3j:crypto:4.3.1'
+    dependency 'org.web3j:eea:4.3.1'
 
     dependency 'org.xerial.snappy:snappy-java:1.1.7.3'
 
+    dependency "tech.pegasys.ethsigner.internal:core:0.2.0"
+    dependency "tech.pegasys.ethsigner.internal:file-based:0.2.0"
+    dependency "tech.pegasys.ethsigner.internal:signing-api:0.2.0"
```

Comment:
nit: should empty line between signing-api and the plugin-api dependencies as these are from different projects